api/HTMLElement.json

```diff
@@ -462,6 +459,81 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "dialog_elements": {
+          "__compat": {
+            "description": "<code>beforetoggle</code> event fires at dialog elements",
```

Comment:
Note, all descriptions like this because it makes them readable if I pull the subfeature into HTMLElementDialog.

Reply:
Usually I would say that we can assume the parent feature as context, and then this description should only be "Fires at dialog elements" and both the `mdn_url` and `spec_url` would be omitted.

For now we can probably make an exception here, but we should eventually solve this in the visualization layer (i.e. the BCD table).

/cc @Elchi3

Reply:
OK. As long as it is good for now. Thank you.

Reply:
FYI, this is for docs PR https://github.com/mdn/content/pull/36833